### ISSUE TITLE
[Bug]:unimo模型选择advertisegen进行训练，执行到第三个循环，没有报错直接退出

### ISSUE THREAD

**Issue by jiweizhangxu** [PaddlePaddle/PaddleNLP#5036]
### 软件环境

```Markdown
- paddlepaddle-gpu: 2.4.1
- paddlenlp: 2.5.0
```

### 重复问题

- [X] I have searched the existing issues

### 错误描述

```Markdown
for inputs in train_data_loader时，到第四个循环 logits = model(*inputs[:-1])加载时间变长，之后跳出程序且没有任何报错出现，请问是什么原因？
下图为debug图片，再往下执行直接就会退出
```

### 稳定复现步骤 & 代码

![错误](https://user-images.githubusercontent.com/88191580/222076805-6839e614-3b2e-45a7-a35c-8a6b0a9d93d7.png)

/root/anaconda3/envs/paddle_fei_new_1/bin/python /usr/local/zx/12345_for_baogao/unimo_advertisegen/run_gen.py
/root/anaconda3/envs/paddle_fei_new_1/lib/python3.9/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
[2023-03-01 15:34:49,166] [    INFO] - Model config UNIMOConfig {
  "attention_probs_dropout_prob": 0.1,
  "bos_token_id": 1,
  "eos_token_id": 3,
  "hidden_act": "relu",
  "hidden_dropout_prob": 0.1,
  "hidden_size": 768,
  "initializer_range": 0.02,
  "intermediate_size": 3072,
  "mask_token_id": 3,
  "max_position_embeddings": 513,
  "model_type": "unimo",
  "normalize_before": false,
  "num_attention_heads": 12,
  "num_hidden_layers": 12,
  "pad_token_id": 0,
  "paddlenlp_version": null,
  "type_vocab_size": 4,
  "unk_token_id": 17963,
  "vocab_size": 18000
}

-----------  Configuration Arguments -----------
batch_size: 16
beta1: 0.9
beta2: 0.98
dataset_name: advertisegen
decode_strategy: beam_search
device: gpu
do_predict: True
do_train: True
epochs: 10
epsilon: 1e-06
learning_rate: 5e-05
length_penalty: 1.2
logging_steps: 10
max_dec_len: 20
max_grad_norm: 1.0
max_seq_len: 512
max_target_len: 30
max_title_len: 30
min_dec_len: 3
model_name_or_path: unimo-text-1.0
num_beams: 6
num_return_sequences: 1
output_path: ./predict.txt
predict_file: ../unimo-text/data/predict_1.json
save_dir: ./checkpoints
save_steps: 1000
seed: 1
temperature: 1.0
top_k: 0
top_p: 1.0
train_file: ../unimo-text/data/wait_data_2.json
warmup_proportion: 0.02
weight_decay: 0.01
------------------------------------------------
W0301 15:34:52.443105 12987 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 3.7, Driver API Version: 10.2, Runtime API Version: 10.2
W0301 15:34:52.450660 12987 gpu_resources.cc:91] device: 0, cuDNN Version: 7.6.
[2023-03-01 15:34:53,340] [    INFO] - All model checkpoint weights were used when initializing UNIMOLMHeadModel.

[2023-03-01 15:34:53,340] [    INFO] - All the weights of UNIMOLMHeadModel were initialized from the model checkpoint at unimo-text-1.0.
If your task is similar to the task the model of the checkpoint was trained on, you can already use UNIMOLMHeadModel for predictions without further training.
[2023-03-01 15:34:53,341] [    INFO] - Already cached /root/.paddlenlp/models/unimo-text-1.0/unimo-text-1.0-vocab.txt
[2023-03-01 15:34:53,358] [    INFO] - tokenizer config file saved in /root/.paddlenlp/models/unimo-text-1.0/tokenizer_config.json
[2023-03-01 15:34:53,359] [    INFO] - Special tokens file saved in /root/.paddlenlp/models/unimo-text-1.0/special_tokens_map.json

Epoch 1/10

Process finished with exit code 134 (interrupted by signal 6: SIGABRT)

**Comment by jiweizhangxu** [comment]
数据集问题，已解决
